pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -139,6 +139,22 @@ func toApiReadOnly(src bool) *ReadOnly {
 	return nil
 }
 
+// Convert_v1_Agent_To_api_Channel creates the channel for guest agent communication
+func Convert_v1_Agent_To_api_Channel(source *v1.GuestAgent, channel *Channel) error {
+
+	channel.Type = "unix"
+	channel.Source = ChannelSource{
+		Mode: "bind",
+		Path: DefaultUnixSocketPath,
```

Comment:
I do not think it is a must, normally user should not care where the unix socket is provided. 
Moreover, I have provided this path only because the code is written in a way I can not omit the field. When the field is omitted the libvirt handles the path automatically.

Reply:
> I have provided this path only because the code is written in a way I can not omit the field

I would suggest to change our xml mapping then and not set the path.